api/Element.json

```diff
@@ -2874,7 +2888,7 @@
           }
         }
       },
-      "onfullscreenerror": {
+      "onerror": {
```

Comment:
drive by comment: why this change? onfullscreenerror -> onerror?

Reply:
Accidental delete key press that I thought I caught but didn't. Fixing in an upcoming PR momentarily.